apps/frontend/components/ui/chart.tsx

```
@@ -52,71 +52,71 @@ function useChart() {
-            ([theme, prefix]) => `
+    const colorConfig = Object.entries(config).filter(
+        ([, config]) => config.theme || config.color,
+    );
+
+    if (!colorConfig.length) {
+        return null;
+    }
+
+    return (
+        <style
+            dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)